cardano-node/app/cardano-cli.hs

```diff
@@ -122,87 +109,22 @@ parseClientCommand =
       <*> parseFilePath    "issuer-key"               "The genesis key that supposedly delegates."
       <*> parseFilePath    "delegate-key"             "The operation verification key supposedly delegated to."
     ])
+  <|> subparser
+  (mconcat
+    [ commandGroup "Transactions"
+    , command' "submit-tx"                      "Submit a raw, signed transaction, in its on-wire representation." $
+      SubmitTx
+      <$> parseTopologyInfo "PBFT node ID to submit Tx to."
+      <*> parseFilePath    "tx"                       "File containing the raw transaction."
+      <*> parseCommonCLI
+    ])
 
-parseTestnetBalanceOptions :: Parser Genesis.TestnetBalanceOptions
+parseTestnetBalanceOptions :: Parser TestnetBalanceOptions
 parseTestnetBalanceOptions =
-  Genesis.TestnetBalanceOptions
+  TestnetBalanceOptions
   <$> parseIntegral        "n-poor-addresses"         "Number of poor nodes (with small balance)."
   <*> parseIntegral        "n-delegate-addresses"     "Number of delegate nodes (with huge balance)."
   <*> parseLovelace        "total-balance"            "Total balance owned by these nodes."
   <*> parseLovelacePortion "delegate-share"           "Portion of stake owned by all delegates together."
   <*> parseFlag            "use-hd-addresses"         "Whether generate plain addresses or with hd payload."
 
-parseLovelace :: String -> String -> Parser Lovelace
```

Comment:
@deepfire   These parsers also need to be put back here as they are only used in `parseClientCommand`

Reply:
I was hoping that some of those parsers would come across as at least somewhat generic : -)

Reply:
Done!